src/jumio-api/jumio-api.service.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Injectable } from '@nestjs/common';
+import { KycDetails } from '../jumio-kyc/kyc.service';
+
+@Injectable()
+export class JumioApiService {
+  async createAccountAndTransaction(): Promise<KycDetails> {
```

Comment:
Will create api interaction in subsequent PR

Reply:
@dgca  didn't you say we needed something else from the returned payload? Is it `web->href`?Here is an example payload:
```
{
    "timestamp": "2023-02-23T18:06:19.894Z",
    "account": {
        "id": "af7...1d1"
    },
    "web": {
        "href": "https://ironfish.web.amer-1.jumio.ai/web/v4/app?authorizationToken=eyJ...i0g&locale=en"
    },
    "sdk": {
        "token": "eyJ...i0g"
    },
    "workflowExecution": {
        "id": "0a...686",
        "credentials": [
            {
                "id": "315...783",
                "category": "ID",
                "allowedChannels": [
                    "WEB",
                    "API",
                    "SDK"
                ],
                "api": {
                    "token": "eyJ...i0g",
                    "parts": {
                        "front": "https://api.amer-1.jumio.ai/api/v1/accounts/af7...1d1/workflow-executions/0a2...686/credentials/315...783/parts/FRONT",
                        "back": "https://api.amer-1.jumio.ai/api/v1/accounts/af7...1d1/workflow-executions/0a2...5686/credentials/315...783/parts/BACK"
                    },
                    "workflowExecution": "https://api.amer-1.jumio.ai/api/v1/accounts/af7...1d1/workflow-executions/0a2...686"
                }
            },
            {
                "id": "6ee...d4b",
                "category": "FACEMAP",
                "allowedChannels": [
                    "WEB",
                    "SDK"
                ]
            },
            {
                "id": "bbe...c96",
                "category": "SELFIE",
                "allowedChannels": [
                    "WEB",
                    "API",
                    "SDK"
                ],
                "api": {
                    "token": "eyJ...i0g",
                    "parts": {
                        "face": "https://api.amer-1.jumio.ai/api/v1/accounts/af7...1d1/workflow-executions/0a2...686/credentials/bbe...c96/parts/FACE"
                    },
                    "workflowExecution": "https://api.amer-1.jumio.ai/api/v1/accounts/af7...1d1/workflow-executions/0a2....b86"
                }
            }
        ]
    }
}